specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2020-12-01-preview/examples/ExportCreateOrUpdateByBillingAccount.json

```diff
@@ -0,0 +1,128 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01",
```

Comment:
Api version needs to be 2020-12-01-preview for here and rest of the places

Reply:
Duh! Thank you.